headshots.py

```diff
@@ -29,13 +36,11 @@
         # ESC pressed
         print("Escape hit, closing...")
         break
-    elif k%256 == 32:
+    if k%256 == 32:
         # SPACE pressed
-        img_name = "dataset/"+ name +"/image_{}.jpg".format(img_counter)
+        img_name = DATASET_FOLDER+"/"+ NAME +"/image_{}.jpg".format(img_counter)
```

Comment:
This could be better formatted as an f-string since oyu have multiple variables in a string.

eg - 

```
menu = ("eggs", "spam", 42.4)

f_string_order = f"{menu[0]} and {menu[1]}: {menu[2]:0.2f} ¤"
```